frontend/src/components/Utils/temp-auth-utils.js

```diff
@@ -1,58 +1,53 @@
 /**
  * @fileoverview This is a temporary file that is used to implement 'fake'
  * versions of the Auth utility functions used in the ViewTrips components.
+ * As of right now, the only 'real' auth function used is `getCurUserEmail()`
+ * but a fake user uid is still used due to the lack of a set of functions
+ * that can convert  user email <--> user uid.
  *
  * TODO(Issue #55): Remove this whole file function and replace any imports to
  *                 this file with Auth utils.
  */
-
-
-/**
- * Temporary hardcoded function that returns the current users email.
- *
- * @return {string} Hardcoded user email string.
- */
-export function getCurUserEmail() {
-  return 'matt.murdock';
-}
+import authUtils from '../AuthUtils';
 
 /**
- * Temporary hardcoded function that returns the current users uid.
+ * Temporary 'fake'/hardcoded function that returns the current users uid.
  *
- * @return {string} Hardcoded user uid string.
+ * @return {string} Fake user uid string the form '_userEmail_'.
  */
 export function getCurUserUid() {
-  return getUserUidFromUserEmail(getCurUserEmail());
+  return getUserUidArrFromUserEmailArr([authUtils.getCurUserEmail()]).pop();
 }
 
 /**
- * Temporary hardcoded function that returns the user's uid given the user's
- * email.
+ * Temporary 'fake'/hardcoded function that returns the an array of 'fake' user
+ * uids given an array of user emails.
  *
- * @param {string} userEmail A users email.
- * @return {string} The 'fake' uid associated with the user email that is
- *     created with the form '_`userEmail`_'.
+ * @param {!Array<string>} userEmailArr Array of user email strings.
+ * @return {!Array<string>} Array of fake uuids corresponding to the user
```

Comment:
Are these uuids or uids? uid seems to be used more frequently throughout these comments. Make sure to be consistent throughout the rest of the code for whichever this is.

Reply:
They are user uids or uuids for short. Because we use userEmail I will switch all uuid to user uid.